lib/src/sep/0099/hotswap.dart

```diff
@@ -0,0 +1,173 @@
+// Copyright 2020 The Stellar Flutter SDK Authors. All rights reserved.
+// Use of this source code is governed by a license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+
+import 'package:http/http.dart' as http;
+import 'package:stellar_flutter_sdk/stellar_flutter_sdk.dart';
+
+class Hotswap {
+  late String _hotswapServerAddress;
```

Comment:
We're already in the hotswap class so the context is already clear.
```suggestion
  late String _serverAddress;
```

Reply:
Done.